.github/scripts/end2end/configs/zenko.yaml

```diff
@@ -123,9 +127,6 @@ spec:
       azure:
         archiveTier: "hot"
         restoreTimeout: "15s"
-  scuba:
-    logging:
-      logLevel: debug
```

Comment:
should we not keep scuba log level to `debug` ? (but move it above, avoid having twice the same key)

Reply:
Not needed. It was done before for testing purposes...